app/apollo/apollo-octopus-public/src/main/graphql/com/hedvig/android/apollo/octopus/graphql/claimflow/FragmentClaimFlowStepFragment.graphql

```diff
@@ -90,21 +90,48 @@ fragment FlowClaimSingleItemStepFragment on FlowClaimSingleItemStep {
 
 fragment FlowClaimSingleItemCheckoutStepFragment on FlowClaimSingleItemCheckoutStep {
   id
-  price {
+  singleItemStep {
+    ...FlowClaimSingleItemStepFragment
+  }
+  availableCheckoutMethods {
+    id
+    ...CheckoutMethodFragment
+  }
+  compensation {
+    ...CompensationFragment
+  }
+}
+
+fragment CompensationFragment on FlowClaimSingleItemCheckoutCompensation {
```

Comment:
This fragment only seems to be used in this one place. I don't think we have any rules or something like that on how to usually approach this, but do you think it'd make sense here to inline this at the one place where it's called? So that we do not introduce a new fragment here which might come with some mental overhead of what this new type is about?
Or do you think that it helps with keeping it easier to work with in the code?

I just kinda remember sometime in the past getting rid of a bunch of fragments which were used only in one place and it typically made things a bit easier to reason about.
But I do not have a very strong opinion about it I think, I just believe that if I wrote it I would try out both and try to see which one feels nicer. 

Or do we perhaps expect this to in the future be used in more than one of these steps?